doc-content/enterprise-only/processes/process-instance-migration-installing-service-proc.adoc

```diff
@@ -58,4 +58,12 @@ kieservers:
 ----
 $ java -jar {PRODUCT_FILE}-process-migration-service-standalone.jar -s./config.yml
 ----
+. To enable Thorntail to auto detect a JDBC driver, add the `JAR` file with the driver name to the `thorntail.classpath` system property. For example, execute the following command:
++
+[source,subs="attributes+"]
+----
+$ java -Dthorntail.classpath=./h2-{PRODUCT_FILE}-process-migration-service-standalone.jar -s./config.yml
+----
++
+NOTE: The `h2` JDBC driver is included by default. You can use different JDBC drivers as per your requirements to connect to the external databases.
```

Comment:
[style] You can simplify by removing "as per your requirements".
[technical] Should it be "different" or "multiple"?
[style] Remove the article for "external databases".

s/NOTE: The `h2` JDBC driver is included by default. You can use different JDBC drivers as per your requirements to connect to the external databases./NOTE: The `h2` JDBC driver is included by default. You can use different JDBC drivers to connect to different external databases.